lib/puppet/type/sensu_resources.rb

```diff
@@ -131,6 +131,9 @@ def check_agent_entity_config(resource)
     if resource[:key] == 'sensu.io/managed_by'
       return false
     end
+    if resource[:config] == 'subscriptions' && resource[:value] == "entity:#{resource[:entity]}"
```

Comment:
Please add a comment here explaining the why and link to issue/pr.

Reply:
comment added